hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/NioAsyncSocket.java

```diff
@@ -247,24 +251,6 @@ public int getTcpKeepaliveProbes() {
         }
     }
 
-    @Override
```

Comment:
SO_LINGER doesn't play nice with nonblocking sockets because either the behavior is undefined or it can lead to blocking. So decided to remove them.